packages/react-storage/src/components/StorageBrowser/views/LocationsView/useLocationsView.ts

```diff
@@ -89,6 +91,10 @@ export function useLocationsView(
     return result.slice(start, end);
   }, [range, result]);
 
+  const filteredItems = React.useMemo(() => {
```

Comment:
filtering in the view for now, i'll follow up with list locations integration once the adapters support the new interface.